Core/NoOpRunner.Core/Shapes/StaticShape.cs

```diff
@@ -4,15 +4,15 @@
 
 namespace NoOpRunner.Core.Shapes
 {
-    public class Platform : BaseShape
+    public abstract class StaticShape : BaseShape
     {
         public int BottomPosY { get; set; } // Bottom of x platform
         public int TopPosY { get; set; } // Max height of x platform
 
         public int LastPosX = 0,
                    LastPosY = 0;
 
-        public Platform(int centerPosX, int centerPosY, int bottomPosY, int topPosY) : base(centerPosX, centerPosY)
+        public StaticShape(int centerPosX, int centerPosY) : base(centerPosX, centerPosY)
```

Comment:
Currently, the platform generation is f*cked. Revert the ctor changes and set `BottomPosY` and `TopPosY` properly here (and make sure to forward the args properly in child ctors). Right now, those two are not set, so a platform is always generated as a straight line.

Reply:
Fixed